src/components/enter-email/tests/enter-email-integration.test.ts

```diff
@@ -8,23 +8,50 @@ import {
   HTTP_STATUS_CODES,
   PATH_NAMES,
 } from "../../../app.constants";
+import nock = require("nock");
+import { ERROR_CODES } from "../../common/constants";
 import { CheckReauthServiceInterface } from "../../check-reauth-users/types";
 import { AxiosResponse } from "axios";
 import { createApiResponse } from "../../../utils/http";
 import { DefaultApiResponse } from "../../../types";
-import nock = require("nock");
+
+const REDIRECT_URI = "https://rp.host/redirect";
 
 describe("Integration::enter email", () => {
   let token: string | string[];
   let cookies: string;
   let app: any;
   let baseApi: string;
+  let checkReauthUsersService: any;
+  let stubConfig: { statusCode: number; shouldStub: boolean };
+
+  function setupCheckReauthUsersStub() {
```

Comment:
I'm not sure this function makes sense for a couple of reasons:

Given it's an integration test, do we instead want to be testing the integration with the reauth users service, and using nock to capture the http call?

If not, this function as is looks like it could be simplified in a few ways:

* add the relevant params to the function so that we're not reading from external state - I can't see any reason here why we need the stub config set as a let outside of this
* When would we not want to stub (`shouldStub` = false?) To me that branch doesn't really make any sense

Reply:
I've removed this as its not used.